jar/jar_test.go

```diff
@@ -102,18 +102,21 @@ func BenchmarkParse(b *testing.B) {
 	}
 }
 
-func TestYARARule(t *testing.T) {
```

Comment:
Can you modify the tests instead of deleting it?

Reply:
done